### ISSUE TITLE
For circular graphics editing, can you edit according to its radius?

### ISSUE THREAD

**Issue by Thunder7991** [JamesLMilner/terra-draw#184]
![1706672555728](https://github.com/JamesLMilner/terra-draw/assets/61643801/7ff1091f-5d6e-449b-804a-3f75c97e182d)
At present, only multi -point operations are supported, editing is not very convenient

**Comment by JamesLMilner** [comment]
Hey @Thunder7991 - thanks for raising, I think this has also been raised as a feature request in #183, so that the shape can be maintained in select mode when editing geometries from Rectangle/Circle mode.

I like this idea and will attempt to add it soon! Will share back when it's implemented.

**Comment by JamesLMilner** [comment]
I will close this in favour of https://github.com/JamesLMilner/terra-draw/issues/183 and track the ability to maintain the feature shape in select mode there. Feel free to share any thoughts on the other issue. Thanks again!

**Comment by JamesLMilner** [comment]
This is available in `0.0.1-alpha.56` as the `maintainShapeFrom` parameter in the Select parameters, `feature` -> `coordinate` -> `maintainShapeFrom` 🎉

Heres an example:

```javascript
				new TerraDrawSelectMode({
					flags: {
				
						polygon: {
							feature: {
								draggable: true,
								rotateable: true,
								scaleable: true,
								coordinates: {
									midpoints: true,
									draggable: true,
									deletable: true,
									maintainShapeFrom: 'center'
								},
							},
						},
```
